includes/news/v2/mail/base.php

```diff
@@ -5,7 +5,6 @@
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="format-detection" content="telephone=no">
-	<link rel="stylesheet" type="text/css" href="https://s3.amazonaws.com/web.ucf.edu/email/common-assets/stylesheet.css">
 	<title>Good
```

Comment:
The President's email markup we use includes webfonts both with this stylesheet _and_ with inline `@font-face` declarations--if webfonts still don't work with this update, consider re-adding this stylesheet.